x-pack/plugins/apm/dev_docs/local_setup.md

```diff
@@ -17,14 +17,35 @@ To access an Elasticsearch instance that has live data you have three options:
 ```
 yarn es snapshot
 ```
+*Note: Elasticsearch will be available on http://localhost:9200*
+
+
+**Start Kibana**
```

Comment:
How does this relate to the Start Kibana section at the top?

Reply:
hm, good point.it's confusing. I should rename it to "Customizing `config/kibana.dev.yml`"  or something similar.